test/protobuf/decoder_test.exs

```diff
@@ -126,7 +126,7 @@ defmodule Protobuf.Decoder.Test do
       use Protobuf, from: Path.expand("../proto/mumble.proto", __DIR__)
     end
 
-    msg = MumbleProto.Authenticate.new(username: "bitwalker")
+    msg = MumbleProto.MumbleProto.Authenticate.new(username: "bitwalker")
```

Comment:
I just tried that and I can't even figure out the name of the module that's getting created. It's not `Basic`, nor `NotMyTopLevelModule`. I'm kinda confused actually. Here is what I see:

```elixir
iex(14)> NotMyTopLevelModule.defs
[{{:msg, :Basic},
  [%Protobuf.Field{fnum: 1, name: :basic, occurrence: :required, opts: [],
    rnum: 2, type: :string}]}]
```

Looks like it's `:Basic` without the `:Elixir.` in front of it but that doesn't work either.

Also it is possible in elixir to define modules without the `Elixir` on the front. That's just the syntactic sugar that adds that when you use uppercase. For example:

```elixir
iex(16)> defmodule :NotElixir do
...(16)> def funky do
...(16)> "hello"
...(16)> end
...(16)> end
{:module, :NotElixir,
 <<70, 79, 82, 49, 0, 0, 4, 124, 66, 69, 65, 77, 69, 120, 68, 99, 0, 0, 0, 128, 131, 104, 2, 100, 0, 14, 101, 108, 105, 120, 105, 114, 95, 100, 111, 99, 115, 95, 118, 49, 108, 0, 0, 0, 4, 104, 2, ...>>,
 {:funky, 0}}
iex(17)> :NotElixir.funky
"hello"
iex(18)> Elixir.NotElixir.funky
** (UndefinedFunctionError) undefined function: NotElixir.funky/0 (module NotElixir is not available)
    NotElixir.funky()
```